infrastructure.go

```diff
@@ -15,16 +18,32 @@ func applyInfraDefinition(c *cli.Context) error {
 		fmt.Println("Please enter valid filename")
 		return nil
 	}
-	fmt.Println("Trying to create or update infrastructure using the yaml=",
+	gcpProjectName := c.String("gcp-project")
+	gcpRegionName := c.String("gcp-region")
+	fmt.Println("Trying to create or update infrastructure using the given yaml=",
 		getColoredText(filePath, color.FgCyan))
 
 	createOrUpdateInfraURL := GetUrlWithQueryParams("", baseURL, INFRA_ENDPOINT, map[string]string{
 		"accountIdentifier": cliCdRequestData.Account,
 	})
 	var content = readFromFile(c.String("file"))
+	if isGcpInfraYaml(content) {
```

Comment:
I see this pattern used all over the Harness-CLI repo, which is not good.
We should use switch case because in case of connector cmd these if's are ever increasing.

Reply:
refactored it.